test/sharness/lib/test-lib.sh

```diff
@@ -280,6 +280,11 @@ test_launch_ipfs_daemon() {
   '
 }
 
+test_launch_ipfs_daemon_without_network() {
+  test_launch_ipfs_daemon --offline "$@"
+  # FIXME: Check if this is the right way to concatenate arguments.
```

Comment:
Blocking.

Reply:
(We can also just drop the argument processing as all calls to `test_launch_ipfs_daemon_without_network` right now don't use them; they were just passing the `--offline` flag.)

Reply:
`"$@"` expands to `"$1" "$2"` and so on (skipping `$0`), should be ok:
https://www.gnu.org/savannah-checkouts/gnu/bash/manual/bash.html#index-_0040